### PR TITLE
Returning indicator for missing HAProxy backends.

### PR DESCRIPTION
**Pull request by menivaitsi**
https://ocd.appscale.com:8080/job/Daily%20Build/1351/

### CODE REVIEW

**Comment by menivaitsi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1358/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -5874,27 +5874,20 @@ def get_all_stats(secret)
...
-                Djinn.log_warn("Try: #{tries} Unforseen exception: #{exception} \nBacktrace: #{backtrace} \n")
-                Kernel.sleep(SMALL_WAIT)
-                tries -= 1
-                retry
-              else
+                }
+              rescue => exception
+                backtrace = exception.backtrace.join("\n")
+                message = "Unforseen exception: #{exception} \nBacktrace: #{backtrace}"
                 Djinn.log_warn("Unable to get application stats and exceeded number of retries: #{message}")
```
Comment (tmarballi) [inline AppController/djinn.rb]: Remove the line 'and exceeded number of retries' as the retry logic is not here anymore.

Comment (cdonati) [inline AppController/djinn.rb]: Is this warning still applicable?

Comment (menivaitsi) [inline AppController/djinn.rb]: @tmarballi will do - thanks.

@cdonati yeah, in case of an unhandled exception while operating on app_info_map.

**Pull request merged**